main/design_documents/ARM_NEON_SVE_BRIDGE.rst

```diff
@@ -0,0 +1,56 @@
+=======================================
+Design Document for ARM_NEON_SVE_BRIDGE
+=======================================
+
+This document describes the design of the intrinsics defined in the ("NEON_SVE bridge")
+<../acle.rst#neon-sve-bridge>. These intrinsics add a
+convienient way to use NEON and SVE vectors interchangeably in C code with
+no performance overhead.
+
+Location
+########
+
+The intrinsics will be in a new header file called ``arm_extensions_bridge.h``.
+
+This header file will define ``__ARM_NEON_SVE_BRIDGE`` as 1.
+
+The intrinsics will be conditionally defined with
+
+.. code:: c
+
+    #if !defined(__ARM_FEATURE_NEON) || !defined(__ARM_FEATURE_SVE)
+        #error "arm_extensions_bridge.h requires -march=+simd+sve"
+    #endif
+    #define __ARM_NEON_SVE_BRIDGE 1
+    //define intrinsics
+
+`__ARM_FEATURE_NEON` is used as it is aarch64 specific, and SVE is not
+supported in the aarch32 state.
+
+Suggestions & Considerations
+############################
+
+Why not enable the extraction of any 128-bit subvector in an SVE vector?
+------------------------------------------------------------------------
+
+Extracting other 128-bit subvectors would go against the no-op design.
+This functionality is already possible with the ACLE intrinsic `svdupq_lane`.
+
+Why not include 64-bit Vectors?
```

Comment:
The question is not directly answered (the "why") by the below paragraph.

Reply:
The direct answer is just that there isn't enough interest currently right?

Reply:
Yes.